app/src/main/res/values/strings.xml

```diff
@@ -2535,4 +2535,7 @@ the Select` button. When clicked it closes all the previously selected tabs -->
     <!-- Tooltip for the battery level when using a tethered device instead of a headset -->
     <string name="tray_status_phone">Phone: %1$s</string>
 
+    <!-- Instruct the user to connect their Vision Glass -->
+    <string name="vision_glass_disconnected_message">Please connect Vision Glass</string>
```

Comment:
I guess we can move this to vision glass' strings.xml ?

Reply:
Done.